src/pages/en/guides/typescript.md

```diff
@@ -82,6 +83,18 @@ const { greeting = 'Hello', name } = Astro.props
 <h2>{greeting}, {name}!</h2>
 ```
 
+## Type checking
+
+To see type errors in your editor please make sure that you have [Astro extention](/en/editor-setup/). Please note that `astro start` and `astro build` will not run any type checking, they just transpile the code with esbuild. To make sure that typescript errors prevent your code from building you can change your "build" script in `package.json`:
+
+```diff
+-    "build": "astro build",
++    "build": "astro check && tsc --noEmit && astro build",
+```
+
+:::note
+`astro check` command only checks types within `.astro` files, and `tsc --noEmit` only checks types within `.ts` and `.tsx` files.
```

Comment:
```suggestion
`astro check` only checks types within `.astro` files, and `tsc --noEmit` only checks types within `.ts` and `.tsx` files.
```
removed "command" because we can be briefer in the note, and both of these commands should be referenced the same way to be consistent. We did mention "commands" in the paragrah above, so I think we can just be shorter, and keep them the same, here.